frontend/src/routes/+layout.svelte

```diff
@@ -62,6 +64,15 @@
     <a href="/login" class="unstyled-link">Login</a>
     <div class="divider"></div>
     <a href="/login" class="unstyled-link">Sign Up</a>
+    <div class="divider"></div>
+    <a
+      href="https://github.com/retrixe"
+      target="_blank"
+      rel="noopener noreferrer"
+      class="unstyled-link last-link"
+    >
+      <GitHubImage class="github-image" viewBox="0 0 98 96" height="28" width="28" />
```

Comment:
Getting a compile error
```suggestion
      <GitHubImage className="github-image" viewBox="0 0 98 96" height="28" width="28" />
```